api/src/test/kotlin/edu/wgu/osmt/collection/CollectionControllerTest.kt

```diff
@@ -124,10 +103,9 @@ internal class CollectionControllerTest @Autowired constructor(
             author = "newAuthor")
         val collection = collectionRepository.create(name = "name", user = "user", email = "j.chavez@wgu.edu", description = "description")
         collection!!.status = PublishStatus.Archived
+        collection!!.archiveDate = LocalDateTime.now()
```

Comment:
IDEA reports that collection!! isn't needed on line 106

Reply:
Yeah, the assertion is already in line 105, let me remove it.